### DISCUSSION TITLE
Thanks for this amazing plugin.

### DISCUSSION THREAD

**Discussion by nasir41** [exodrifter/unity-python#1] [Q&A] [answered]
How I can Import required libs in unity?
in my .py file I my code require following 3 libs so how I can import it in unity so that ruby compiler use it?

require "rest-client"
require "colorize"
require "json"

**Comment by exodrifter** [comment, marked as answer]
There is no simple way to "import" your python code to Unity. I'm not really sure how to do it myself, but I can suggest reading the IronPython documentation on [Accessing Python code from other .NET code](http://ironpython.net/documentation/dotnet/dotnet.html#accessing-python-code-from-other-net-code) for starters.

Alternatively, you can wrap every function you want to access and then invoke them from C#. From this stackoverflow [example](https://stackoverflow.com/questions/13231913/how-do-i-call-a-specific-method-from-a-python-script-in-c):
```
var pySrc =
@"def CalcAdd(Numb1, Numb2):
    return Numb1 + Numb2";

// host python and execute script
var engine = IronPython.Hosting.Python.CreateEngine();
var scope = engine.CreateScope();
engine.Execute(pySrc, scope);

// get function and dynamically invoke
var calcAdd = scope.GetVariable("CalcAdd");
var result = calcAdd(34, 8); // returns 42 (Int32)

// get function with a strongly typed signature
var calcAddTyped = scope.GetVariable<Func<decimal, decimal, decimal>>("CalcAdd");
var resultTyped = calcAddTyped(5, 7); // returns 12m
```
